CONTRIBUTING.md

```diff
@@ -164,14 +164,14 @@ To follow [Go testing conventions](https://golang.org/pkg/cmd/go/internal/test/)
 
 Our integration tests can be run against either a GKE / Openshift remote cluster. To do so, run `./bin/start` and add the proper flags. 
 
-To deploy OSS / DAP, add the `--oss` / `--dap` flags to the above command. By default, the integration tests run DAP, so no flag is required.
+To deploy Conjur Open Source / Conjur Enterprise, add the `--oss` / `--dap` flags to the above command. By default, the integration tests run Conjur Enterprise, so no flag is required.
```

Comment:
you sometime write `Conjur Open Source / Conjur Enterprise` and sometimes `Conjur Open Source/Enterprise`. Can you ask Shuli (can't tag her for some reason) for the right one and stick to it?

Reply:
Done